modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -71,31 +84,29 @@ const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 
 				const balloonToolbars = editor.balloonToolbars;
 
-				balloonToolbars.create({
-					buttons: editorConfig.toolbarText,
-					cssSelector: '*',
-				});
-
-				balloonToolbars.create({
-					buttons: editorConfig.toolbarImage,
-					priority:
-						window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH,
-					widgets: 'image,image2',
-				});
-
-				balloonToolbars.create({
-					buttons: editorConfig.toolbarTable,
-					cssSelector: 'td',
-					priority:
-						window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH,
-				});
-
-				balloonToolbars.create({
-					buttons: editorConfig.toolbarVideo,
-					priority:
-						window.CKEDITOR.plugins.balloontoolbar.PRIORITY.HIGH,
-					widgets: 'videoembed',
-				});
+				const toolbarPanelConfigs = Object.fromEntries(
+					Object.entries(editorConfig).filter(([key, _]) =>
+						/^toolbar.*$/.test(key)
```

Comment:
I saw we have a pattern starting with `toolbar` + _toolbarType_, So I decided to use a regexp.

~~May I wrong about this decision of using RegEX? For sure. But it helped me a lot to filter. ~~

I can use [`.startsWith`](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) but I'm not sure if it's okay to check the literacy of the key. 

What do you think @nhpatt and @markocikos?

Reply:
I'll move this PR to draft to avoid ppl acidentally run `ci:forward`

Reply:
> I can use .startsWith but I'm not sure if it's okay to check the literacy of the key.

.startsWith works well. Replacing it.

Reply:
See: c00c117

Reply:
> I can use .startsWith but I'm not sure if it's okay to check the literacy of the key.

You are doing the same with the regexp, right? but more legible.

Reply:
>IMHO You are doing the same with the regexp, right? but more legible now.

Yas! :)